Translations/zh-Hans.lproj/Credits.html

```diff
@@ -86,7 +86,7 @@
     
     
     <!-- START WARRANTY -->
-    <p class="warranty">THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.</p>
+    <p class="warranty">该软件由版权持有人和贡献者提供,如“是”和任何明示或暗示的保证,包括但不限于适销性和适合特定目的的默示保证被驳回.在任何情况下,版权持有人或贡献者均不对任何直接、间接、附带、特殊、惩戒性或后果性损害(包括但不限于替代货物或服务的采购;使用数据或利润的损失;或业务中断)承担责任,无论其原因如何,也不论其责任理论是否在合同中,因使用本软件而产生的严格责任或侵权行为(包括疏忽或其他),即使已告知此类损害的可能性.</p>
```

Comment:
I don't recommend translating the warranty notice unless this translation has been proof-read by a lawyer. The phrasing is not very natural, either.

Reply:
totally agree